artemis/modules/nuclei.py

```diff
@@ -88,6 +89,13 @@ def _get_links(self, url: str) -> List[str]:
         links = links[: Config.Modules.Nuclei.NUCLEI_MAX_NUM_LINKS_TO_PROCESS]
         return links
 
+    def _strip_query_string(self, url: str) -> str:
+        url_parsed = urllib.parse.urlparse(url)
+        url_parsed_dict = url_parsed._asdict()
```

Comment:
why though, you can easily change fields via `._replace()`, this seems overkilly